tutorials/setup-your-development-environment.md

```diff
@@ -0,0 +1,67 @@
+# Setup your development environment
+
+The following section walks you through the necessary steps to setup your environment.
+It is recommended to spin up a clean Ubuntu VM for these steps as it is always good to perform experiments in an isolated development environment.
+The steps outlined [here](https://juju.is/docs/sdk/set-up-your-development-environment#heading--set-up-an-ubuntu-vm-with-multipass) explain how to do this.
+
+## Create a project directory
+
+Create a new project directory with
+```bash
+mkdir ~/go-dqlite-demo
+cd ~/go-dqlite-demo
+```
+
+This will create a new folder called `go-dqlite-demo` in your home directory and opens it. All project files will be put in this directory.
```

Comment:
```suggestion
This will create a new folder called `go-dqlite-demo` in your home directory and enters it. All project files will be put in this directory.
```